WordPress/src/main/java/org/wordpress/android/ui/comments/unified/UnifiedCommentListViewModel.kt

```diff
@@ -1,55 +1,76 @@
 package org.wordpress.android.ui.comments.unified
 
-import androidx.paging.Pager
-import androidx.paging.PagingConfig
-import androidx.paging.insertSeparators
-import androidx.paging.map
 import kotlinx.coroutines.CoroutineDispatcher
-import kotlinx.coroutines.flow.map
-import org.wordpress.android.fluxc.model.CommentStatus
 import org.wordpress.android.modules.UI_THREAD
-import org.wordpress.android.ui.comments.unified.UnifiedCommentListItem.ClickAction
-import org.wordpress.android.ui.comments.unified.UnifiedCommentListItem.Comment
-import org.wordpress.android.ui.comments.unified.UnifiedCommentListItem.SubHeader
-import org.wordpress.android.ui.comments.unified.UnifiedCommentListItem.ToggleAction
+import org.wordpress.android.util.DateTimeUtilsWrapper
 import org.wordpress.android.viewmodel.ScopedViewModel
 import javax.inject.Inject
 import javax.inject.Named
 
+/**
+ * Temporarily commented out until migration between Paging 2 and 3 is sorted out.
+ */
 class UnifiedCommentListViewModel @Inject constructor(
+    private val dateTimeUtilsWrapper: DateTimeUtilsWrapper,
     @Named(UI_THREAD) private val mainDispatcher: CoroutineDispatcher
 ) : ScopedViewModel(mainDispatcher) {
     private var isStarted = false
-
-    // TODO we woiuld like to explore moving PagingSource into the repository
-    val commentListItemPager = Pager(PagingConfig(pageSize = 30, initialLoadSize = 30)) { CommentPagingSource() }
-
-    val commentListItems = commentListItemPager.flow
-            .map { pagingData ->
-                pagingData.map { commentModel ->
-                    val toggleAction = ToggleAction(commentModel.remoteCommentId, this::clickItem)
-                    val clickAction = ClickAction(commentModel.remoteCommentId, this::toggleItem)
-                    Comment(
-                            remoteCommentId = commentModel.remoteCommentId,
-                            postTitle = commentModel.postTitle,
-                            authorName = commentModel.authorName,
-                            authorEmail = commentModel.authorEmail,
-                            body = commentModel.content,
-                            avatarUrl = "",
-                            isPending = commentModel.status == CommentStatus.UNAPPROVED.toString(),
-                            isSelected = false,
-                            clickAction = clickAction,
-                            toggleAction = toggleAction
-                    )
-                }
-                        .insertSeparators { before, after ->
-                            when {
-                                before == null -> SubHeader("Date Sub Header", -1)
-                                // TODO add the logic for determining if subheader is necessary or not
-                                else -> null
-                            }
-                        }
-            }
```

Comment:
Side note that the diff is slightly different here from what was removed in [the other PR](https://github.com/wordpress-mobile/WordPress-Android/pull/14949/files#diff-bb41a4453c1a2ef9ede4980e28a061c4aedc44774ee4434bf93b19bf61e49d9dL36-L90). Probably expected given this part of the code like changed between release and develop, but just wanted to highlight this to double-check that the different code being removed did not have side effects that the original didn't have.

Also, that means that the code that is actually added and commented out on the following lines 18-73 does not match what was removed, so when we get back to this later to "revert this revert" and try to get back to migrating to paging 3, there is a risk that we would just uncomment the lines 18-73 (instead of reverting this PR) and land in a different state of the code that what was expected, introducing regressions and bugs or behavior changes that were fixed by the new code that was _removed_ above but were still present in the code that was _commented_ below